.github/workflows/bump_submodules.yml

```
@@ -0,0 +1,46 @@
+name: Bump Submodule
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,4'  # Run at 00:00 on Sunday and Thursday
+  pull_request:
+    branches: ["main"]
+    paths:
+      - ".github/workflows/bump_submodules.yml"
+
+jobs:
+  bump_submodule:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
```

Same here - follow project style
```suggestion
    - name: Checkout repository
      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
```